pages/pricing.vue

```diff
@@ -37,6 +37,62 @@
           <PricingCard :plan="personalYearly" :isHighlighted="true" />
           <PricingCard :plan="businessYearly" />
         </div>
+        <div class="table-responsive mt-5 py-5">
+          <h2 class="comparision__table-heading mb-5">
+            See All Features and Compare Plan
+          </h2>
+          <table class="table text-start">
```

Comment:
I think the `table-responsive` may be the class which is reponsible for scrolling heading. We might need to move it down to the `table` tag.

Reply:
no @piyushsinghania this the way table responsive class is used.

Reply:
you can check the bootstrap official docs